src/12_RefactoringGolf/hole1/kata.ts

```diff
@@ -17,58 +18,58 @@ export class Game {
       throw new Error('Invalid next player');
     }
     //if not first move but play on an already played tile
-    else if (this._board.TileAt(x, y).Symbol != ' ') {
+    else if (this._toto.TileAt(x, y).Symbol != ' ') {
       throw new Error('Invalid position');
     }
 
     // update game state
     this._lastSymbol = symbol;
-    this._board.AddTileAt(symbol, x, y);
+    this._toto.AddTileAt(symbol, x, y);
   }
 
   public Winner(): string {
     //if the positions in first row are taken
     if (
-      this._board.TileAt(0, 0)!.Symbol != ' ' &&
-      this._board.TileAt(0, 1)!.Symbol != ' ' &&
-      this._board.TileAt(0, 2)!.Symbol != ' '
+      this._toto.TileAt(0, 0)!.Symbol != ' ' &&
+      this._toto.TileAt(0, 1)!.Symbol != ' ' &&
+      this._toto.TileAt(0, 2)!.Symbol != ' '
     ) {
       //if first row is full with same symbol
       if (
-        this._board.TileAt(0, 0)!.Symbol == this._board.TileAt(0, 1)!.Symbol &&
-        this._board.TileAt(0, 2)!.Symbol == this._board.TileAt(0, 1)!.Symbol
+        this._toto.TileAt(0, 0)!.Symbol == this._toto.TileAt(0, 1)!.Symbol &&
+        this._toto.TileAt(0, 2)!.Symbol == this._toto.TileAt(0, 1)!.Symbol
       ) {
-        return this._board.TileAt(0, 0)!.Symbol;
+        return this._toto.TileAt(0, 0)!.Symbol;
       }
     }
 
     //if the positions in first row are taken
     if (
-      this._board.TileAt(1, 0)!.Symbol != ' ' &&
-      this._board.TileAt(1, 1)!.Symbol != ' ' &&
-      this._board.TileAt(1, 2)!.Symbol != ' '
+      this._toto.TileAt(1, 0)!.Symbol != ' ' &&
+      this._toto.TileAt(1, 1)!.Symbol != ' ' &&
+      this._toto.TileAt(1, 2)!.Symbol != ' '
     ) {
       //if middle row is full with same symbol
       if (
-        this._board.TileAt(1, 0)!.Symbol == this._board.TileAt(1, 1)!.Symbol &&
-        this._board.TileAt(1, 2)!.Symbol == this._board.TileAt(1, 1)!.Symbol
+        this._toto.TileAt(1, 0)!.Symbol == this._toto.TileAt(1, 1)!.Symbol &&
+        this._toto.TileAt(1, 2)!.Symbol == this._toto.TileAt(1, 1)!.Symbol
       ) {
-        return this._board.TileAt(1, 0)!.Symbol;
+        return this._toto.TileAt(1, 0)!.Symbol;
       }
     }
 
     //if the positions in first row are taken
```

Comment:
This may be the "second row" on this comment

Reply:
coucou